_dashboard/DotNetPerfMonitor/components/dashboard/line_chart.vue

```diff
@@ -1,5 +1,174 @@
 <template>
     <div>
-        DashboardLineChart
+        <LineChart :data="data" />
+
     </div>
-</template>
\ No newline at end of file
+</template>
+
+
+<script setup>
+const props = defineProps({
+    scenario: String
+})
+const path = 'https://raw.githubusercontent.com/G-Research/DotNetPerfMonitor/main/data.csv'
+
+const converted = await useCsvConverter(path)
+const scenario = useAlphaScenario()
+const filtered = useScenarioFilter(converted, 'noop')
+const largeAppCPM = useBenchmarkGrouper(filtered, "LargeAppCPM-142722b")
+const largeAppCPM64 = useBenchmarkGrouper(filtered, "LargeAppCPM-nostaticgraph-142722b")
+const orleans = useBenchmarkGrouper(filtered, "Orleans-eda972a")
+const orchardcore = useBenchmarkGrouper(filtered, "OrchardCore-5dbd92c")
+const nuget = useBenchmarkGrouper(filtered, "NuGetClient-win-d76a117",)
```

Comment:
This should be dynamic so we can add easily new benchmarks without updating the frontend code.